platforms/php/versionsToBuild.txt

```diff
@@ -1,4 +1,7 @@
 # version, sha, gpg keys, dockerfile
 7.2.33, 0f160a3483ffce36be5962fab7bcf09d605ee66c5707df83e4195cb796bbb03a, 1729F83938DA44E27BA0F4D3DBDB397470D12172 B1B44D8F021E4E2D6021E995DC9FF8D3EE5AF27F,
+7.2.34, 409e11bc6a2c18707dfc44bc61c820ddfd81e17481470f3405ee7822d8379903, 1729F83938DA44E27BA0F4D3DBDB397470D12172 B1B44D8F021E4E2D6021E995DC9FF8D3EE5AF27F,
 7.3.21, 4c8b065746ef776d84b7ae47908c21a79e3d4704b86b60d816716b8697c58ce9, CBAF69F173A0FEA4B537F470D66C9593118BCCB6 F38252826ACD957EF380D39F2F7956BC5DA04B5D, 7.3/Dockerfile
-7.4.9, 23733f4a608ad1bebdcecf0138ebc5fd57cf20d6e0915f98a9444c3f747dc57b, 42670A7FE4D0441C8E4632349E4FDC074A4EF02D 5A52880781F755608BF815FC910DEB46F53EA312,
\ No newline at end of file
+7.3.25, c71c00ad03079efb78d1a6b8623ca4f725be697dbd9a46debacbcc9a2475f329, CBAF69F173A0FEA4B537F470D66C9593118BCCB6 F38252826ACD957EF380D39F2F7956BC5DA04B5D, 7.3/Dockerfile
```

Comment:
Have you triggered a success build for php base images and platform binaries.

Reply:
yes ...  you can find the new base image tags in the constant.yaml